carmin.yaml

```diff
@@ -1,6 +1,6 @@
-swagger: '2.0'
+openapi: '3.0.0'
 info:
-  version: '0.2.1'
```

Comment:
Isn't the switch to OpenAPI 3.0.0 a big enough change (it introduce some incompatibilities) to define a 0.3.0 CARMIN version ?

Reply:
I just wrote 0.2.2 as a temporary version just in develop (not to be released), waiting for the other PR (especially the data module) to be merged, and all this package would be the 0.3.0 version.